doc_admin/AdminManual.adoc

```diff
@@ -390,22 +390,53 @@ NOTE: Remember to set the updated database password also in the Open Hospital co
 [#setup]
 ==== 2.1.5 OH - CLIENT configuration
 
-To set up the database connection for Open Hospital, edit and configure the database settings in the **oh.sh / oh.ps1** startup scripts or directly modify the _database.properties_ and _log4j.properties_ configuration files, located in the _oh/rsc/_ resource folder.
+To set up the database connection for Open Hospital, edit and configure the database settings in the **oh.sh / oh.ps1** startup scripts.
 
-Specify, according to the values specified in the installation phase:
+Database settings are:
 
 * the *IP address* of the the MariaDB/MySQL database server hosting the the Open Hospital database. Replace “localhost” with the server IP address (e.g., 192.168.1.100)
 * the *TCP Port Number* of the database server - default is “3306"
 * the *Database name* of the OH installation - default is "oh"
 * the *Database user* of the OH database - default is "isf"
 * the *Database password* associated with the user
 
+Please set them according to the values specified in the installation phase:
+
+in **oh.sh / oh.ps1** scripts:
+
+```
+$script:MYSQL_SERVER="127.0.0.1"
+$script:MYSQL_PORT=3306
+$script:MYSQL_ROOT_PW="tmp2021oh111"
+$script:DATABASE_NAME="oh"
+$script:DATABASE_USER="isf"
+$script:DATABASE_PASSWORD="isf123"
+```
+
+After you can generate the config files: on Linux execute *oh.sh -g*, on Windows launch *oh.bat* and choose the option *g*.
+
+*Alternative method*: you can first generate the config files with the *-g* and after directly modify the _database.properties_ and _log4j.properties_ configuration files, located in the _oh/rsc/_ resource folder, without generating them every time.
+
+Please set them according to the values specified in the installation phase:
+
+in _database.properties_ config file:
```

Comment:
as there are two files to examine; would it make more sense to make a ordered or unordered list to highlight the two files?